src/main/java/com/internet/shop/service/impl/OrderServiceImpl.java

```diff
@@ -0,0 +1,41 @@
+package com.internet.shop.service.impl;
+
+import com.internet.shop.dao.OrderDao;
+import com.internet.shop.lib.Inject;
+import com.internet.shop.lib.Service;
+import com.internet.shop.model.Order;
+import com.internet.shop.model.ShoppingCart;
+import com.internet.shop.service.OrderService;
+import java.util.List;
+
+@Service
+public class OrderServiceImpl implements OrderService {
+    @Inject
+    OrderDao orderDao;
+
+    @Override
+    public Order completeOrder(ShoppingCart shoppingCart) {
+        return orderDao.completeOrder(shoppingCart);
+    }
+
+    @Override
+    public List<Order> getUserOrders(Long userId) {
+        return orderDao.getUserOrders(userId);
+    }
+
+    @Override
+    public Order get(Long id) {
+        return orderDao.get(id)
+                .orElseThrow();
```

Comment:
```suggestion
        return orderDao.get(id).get();
```
we can use just get() here, in case it won't find object, it will throw NoSuchElementException